src/FV/src/math/lu_implementation.F90

```diff
@@ -47,7 +39,8 @@
 !
 
 SUBMODULE (tools_math) lu_implementation
-  IMPLICIT NONE
+    IMPLICIT NONE
+    !! summary: Procedures for matrix LU factorization and solution
```

Comment:
FORD documentation meta data needs to go immediately after module/submodule declaration

Reply:
(i.e. move it up before `IMPLICIT NONE`)

Reply:
Ah, I see now... yes, we should *copy* the summary and category meta-data to the module scope too... I wish FORD could inherit this from the parent thing (i.e. the file)